pkg/models/nda_agreement.go

```diff
@@ -10,7 +10,7 @@ import (
 type NDAAgreement struct {
 	// baseStruct
 	UserID    uuid.UUID  `json:"userID" db:"user_id"`
-	Agreed    bool       `json:"agreed" db:"agreed"`
+	Agreed    bool       `json:"agreed" db:"v2_agreed"`
```

Comment:
It might make sense to return the raw data so we have entries for both fields.
```suggest
V1Agreed bool `json:"v1_agreed" db:"v1_agreed"`
V1AgreedDts *time.Time `json:"v1agreedDts" db:"v1_agreed_dts"`
V2Agreed bool `json:"v2_agreed" db:"v2_agreed"`
V2AgreedDts *time.Time `json:"v2agreedDts" db:"v2_agreed_dts"`
```


Then you can make a computed resolver for the Agreed field that effectively returns the most recent agreement. But, it also leaves the data available in GQL if needed.